workspaces/theme/plugins/theme/src/utils/createComponents.ts

```diff
@@ -541,21 +562,17 @@ export const createComponents = (themeConfig: ThemeConfig): Components => {
       styleOverrides: {
         tabsWrapper: {
           paddingLeft: '0',
-          backgroundColor: general.mainSectionBackgroundColor,
         },
         defaultTab: {
           textTransform: 'none',
-          fontSize: '1rem',
-          fontWeight: '500',
+          fontSize: '0.875rem',
+          fontWeight: '400',
           color: general.disabled,
-          padding: '0.5rem 1rem',
-          '&:hover': {
-            boxShadow: `0 -3px ${general.tabsBottomBorderColor} inset`,
-          },
+          padding: '0 0.25em',
```

Comment:
What do you think about:

```suggestion
          padding: '0.25em 0.5rem',
```

I think that's what is used on https://www.patternfly.org/components/tabs/html/#tabs-as-navigation , or?

Reply:
this looks better. thanks!